setup.py

```diff
@@ -23,31 +23,29 @@
 
 
 def get_requirements():
-    """Parse all packages mentioned in the 'requirements.txt' file."""
-    with open('requirements.txt') as fd:
+    """Parse dependencies from 'requirements.in' file."""
```

Comment:
Provide Descriptive Description and Possible use Type Casting. 

Help someone who will be working on your code to get an idea- What the Heck is happening..!

Reply:
Description is provided.